### PR TITLE
Adds ParadoxSpace's custom items.

### PR DESCRIPTION
**Pull request by AtomicWorm**
Details: https://baystation12.net/forums/threads/approved-paradoxspace-oliver-magent-multiple.5009/

### CODE REVIEW

File: config/custom_items.txt [hunk 1]
```diff
@@ -726,3 +726,21 @@ item_name: Diamond Insignia
...
+{
+ckey: paradoxspace
+character_name: Oliver Magent
+item_path: /obj/item/clothing/head/beret
+item_name: Crown of Magent
+item_icon: olivercrown
+item_desc: A heavy, silver coronet inlaid with sparkling ruby gems. It looks recently polished.
+}
+
+}
```
Comment (mustafakalash) [inline config/custom_items.txt]: You got your brackets backwards.

File: config/custom_items.txt [hunk 2]
```diff
@@ -726,3 +726,21 @@ item_name: Diamond Insignia
...
+ckey: paradoxspace
+character_name: Oliver Magent
+item_path: /obj/item/clothing/suit/storage/hooded/wintercoat
+item_name: Royal Cloak
+item_icon: olivercloak
+item_desc: A thick, red, 'synthfur' hooded cloak. It looks to be insulated enough for a cold day on Yuclite.
+{
```
Comment (mustafakalash) [inline config/custom_items.txt]: Also backwards.

**Review by mustafakalash — CHANGES_REQUESTED** [review]

File: config/custom_items.txt [hunk 3]
```diff
@@ -726,3 +726,21 @@ item_name: Diamond Insignia
...
+item_path: /obj/item/clothing/head/beret
+item_name: Crown of Magent
+item_icon: olivercrown
+item_desc: A heavy, silver coronet inlaid with sparkling ruby gems. It looks recently polished.
+}
+
+}
+ckey: paradoxspace
+character_name: Oliver Magent
+item_path: /obj/item/clothing/suit/storage/hooded/wintercoat
```
Comment (mustafakalash) [inline config/custom_items.txt]: Cloak with pockets?

**Comment by AtomicWorm** [comment]
Well, that was dumb.

File: config/custom_items.txt (hunk 3 above)
Comment (Nerev4r) [reply config/custom_items.txt]: Cloaks can have pockets.

Comment (AtomicWorm) [reply config/custom_items.txt]: It was approved as a child of the winter coat. *shrug

File: config/custom_items.txt [hunk 4]
```diff
@@ -726,3 +726,21 @@ item_name: Diamond Insignia
...
+item_path: /obj/item/clothing/head/beret
+item_name: Crown of Magent
+item_icon: olivercrown
+item_desc: A heavy, silver coronet inlaid with sparkling ruby gems. It looks recently polished.
+}
+
+{
+ckey: paradoxspace
+character_name: Oliver Magent
+item_path: /obj/item/clothing/suit/storage/hooded/wintercoat
```
Comment (mustafakalash) [inline config/custom_items.txt:742]: Cloak with pockets? And also hooded, but no hood?

**Review by mustafakalash — CHANGES_REQUESTED** [review]

File: config/custom_items.txt (hunk 4 above)
Comment (AtomicWorm) [reply config/custom_items.txt:742]: Hey man, I'm just doing the PR the way PS wanted it. I did warn him the hood wouldn't work.

**Review by mustafakalash — APPROVED** [review]
Apparently, pockets are intentional and coat type is necessary.

**Pull request merged**